### ISSUE TITLE
[Fix Suggestions] Floating Edge Check

### ISSUE THREAD

**Issue by sayas01** [osmlab/atlas-checks#405]
**Which Atlas check is the fix suggestion related to?** 

**FloatingEdgeCheck**

Description: https://github.com/osmlab/atlas-checks/blob/dev/docs/checks/floatingEdgeCheck.md

Source Code: https://github.com/osmlab/atlas-checks/blob/dev/src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/FloatingEdgeCheck.java

**Describe more details of suggested fix(es)**

*Suggested Fix 1*: 

- Type: Geometry

- Operation: Delete

- Action: Remove the floating edge

**Is there any code enhancement needed if adding the fix suggestion component?**

No

**Comment by atiannicelli** [comment]
This seems extreme to me to suggest to delete an edge that is floating. If we suggest to delete it then people may blindly delete edges instead of connecting them properly to another edge.

**Comment by Bentleysb** [comment]
I agree with @atiannicelli. I think the fix suggestion case for this check would be to see if there is a nearby Edge and suggest connecting to that. If there are no nearby Edges then the fix would likely be more than we can suggest programmatically.
